src/Order/CommandHandler/AddOrderPaymentCommandHandler.php

```diff
@@ -51,15 +45,17 @@ class AddOrderPaymentCommandHandler extends AbstractOrderHandler
     private $configuration;
 
     public function __construct(
-        EventDispatcherInterface $eventDispatcher,
         FundingSourceTranslationProvider $fundingSourceTranslationProvider,
         PayPalConfiguration $configuration
     ) {
-        $this->eventDispatcher = $eventDispatcher;
         $this->fundingSourceTranslationProvider = $fundingSourceTranslationProvider;
         $this->configuration = $configuration;
     }
 
+    public function __invoke(AddOrderPaymentCommand $command) {
```

Comment:
By the way, if you use the `AsCommandHandler` you shouldn't need to add this `__invoke` method, but again I'm really aware of this module's architecture and context If the command/query buses are used in FO/legacy context this auto-configuration will probably not work

Reply:
I will check if this works on front controllers